packages/graph/hash_graph/lib/graph/src/api/rest/entity.rs

```diff
@@ -147,8 +147,8 @@ async fn get_entity<P: GraphPool>(
 )]
 async fn get_latest_entities<P: GraphPool>(
```

Comment:
Same as above but for line 144 and 
`body = [PersistedEntity]` instead

Reply:
Should be fixed as well